problem2.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('node-fetch');
+
+Array.prototype.populate = function(){
+
+    this.forEach((element,index) => {
+
+        fetch(element)
+        .then(response => this[index] = response)
+        .catch(err => console.log ("Unable to fetch " + element + " " + err));
+    });
+
+
+}
+
+const array = ["https://www.google.com", "https://www.facebook.com"];
+
+array.populate();
+
+setTimeout(() => {
+    console.log(array)
+}, 5000);
```

Comment:
This assignment is not done correctly. Using result of an asynchronous method with setTimeout is not a right way to do it. 
Make the populate function async and then call it using promise or async/await.